### ISSUE TITLE
Chinese DOS/V mode running in DOSBox-X Cannot install windows3.x 

### ISSUE THREAD

**Issue by yahooboby** [joncampbell123/dosbox-x#2695]
Set the language cn.lng in Main and dosv in DOS/V to cn.
Can display Chinese into DOS, but I installed any version of windows3.x
(Including English) are stuck on the installation screen, unable to proceed to the next step.
![4](https://user-images.githubusercontent.com/72782990/126450699-200a6953-caa2-4b5a-882e-542916a64d6e.png)

Close language in Main is empty and DOS/V is off,
Any version of windows 3.x can be installed 
![5](https://user-images.githubusercontent.com/72782990/126450733-63a26697-59cf-4ff0-8076-756e2ec2f0c6.png)

windows3.x download address:
https://winworldpc.com/product/windows-3/31 

Thank you!

This Chinese translation into English is translated by Google.

**Comment by Wengier** [comment]
Yes, the Chinese DOS/V mode is mainly intended to run DOS/V programs and other relatively light-weighted DOS programs. When classic Chinese DOS systems (CCDOS, TW, etc) are running you can also not run heavy-weighted programs. If you want to run general non-DOS/V programs like Windows 3.x installations, then it is strongly recommended to use the **Chinese TTF mode** instead. For Chinese TTF mode, please set "dosv" to "off", and set "country=86,936" and "output=ttf", as well as a TTF font that supports Chinese characters, e.. the simkai.ttf font included with Windows, or a TTF font like [SarasaGothicFixed.ttf](https://github.com/joncampbell123/dosbox-x/raw/master/contrib/fonts/SarasaGothicFixed.ttf) which may look better. Set the TTF font with the "font" config option, e.g. "font=simkai" or "font=SarasaGothicFixed".

Unlike Chinese DOS/V mode, the Chinese TTF mode provides complete DOS emulations just like regular English mode.

A screenshot for the Chinese TTF mode:

![image](https://user-images.githubusercontent.com/8216923/126453043-09d2d80e-5586-4964-817e-f3a470f627c5.png)

**Comment by yahooboby** [comment]
Thanks a lot!
Set according to your requirements, you can install window3.x.
![7](https://user-images.githubusercontent.com/72782990/126456940-27eed3a4-6113-433e-8f06-8cc484ed12d5.png)

But there is a little problem. .
see the picture 
![6](https://user-images.githubusercontent.com/72782990/126456963-64b6b62b-7653-4954-af58-296025b062a1.png)

All dots are in this square, and the same is true when changing N fonts. 
Thank you again for your help!

**Comment by Wengier** [comment]
@yahooboby Yes, it looks like the dots are not included in e.g. the simkai font. But the dots should be rendered correctly if you use the SarasaGothicFixed.ttf font I linked above. Download the font and set ``font=SarasaGothicFixed`` and it should work:

![image](https://user-images.githubusercontent.com/8216923/126458246-7f909b32-87f9-4bec-94a3-1d79585e4247.png)

**Comment by yahooboby** [comment]
Temporarily normal. . No problems were found.
I want to ask if the font you sent is free and free.
I don't want to trouble the law because of this font. .
China strictly regulates fonts, and the use of fonts without consent will be subject to legal procedures. 
thanks

**Comment by Wengier** [comment]
Yes, the Sarasa Gothic (https://github.com/be5invis/Sarasa-Gothic) TTF fonts are a class of TTF fonts supporting both western and CJK characters under the free OFL 1.1 license. I think they look beautiful too. Hope his helps.

**Comment by yahooboby** [comment]
No problem. thank you
